PBuild/Options.pm

```diff
@@ -199,6 +201,8 @@ Important options (see the man page for a full list):
         defaults to the host architecture
   --obs url
         open build service instance for obs:/ type urls
+  --no-clean
+        re-use former build environment
```

Comment:
Please do not mention no-clean in the usage. The man page is enough.

Reply:
done